### PR TITLE
Test: add docker environment for e2e testing

### LINKED ISSUE #11560: [CI] Build a docker container for e2e testing environment
### Current Behavior
Currently, the playwright testing environment is still relying on MesheryCLI, it creates a manual process for dealing with testing
1. Setup the server and frontend using MesheryCLI
2. Run the tests

Another problem is mostly MesheryCLI depends on the new release version, but we also need the testing to run before the release process happens

### Desired Behavior
We want to utilize [playwright web server](https://playwright.dev/docs/test-webserver#configuring-a-web-server) to create an automated process

### Implementation
Build an optimized docker container to make sure it does not get a timeout

### Acceptance Tests
1. It should be nonblocking the testing environment
2. Not take too much time to create
3. Should integrate smoothly with the playwright web server

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
- Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)
- 🧪 [Meshery Test Plan Spreadsheet](https://docs.google.com/spreadsheets/d/13Ir4gfaKoAX9r8qYjAFFl_U9ntke4X5ndREY1T7bnVs/edit#gid=0)

### PR DESCRIPTION
**Pull request by jerensl**
**Notes for Reviewers**
As discussed in the CI meeting, I made separate PR for addressing this issue

This PR fixes #11560

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -32,6 +32,12 @@ docker-playground-build:
 	# This method does NOT require that you have Go, NPM, etc. installed locally.
 	DOCKER_BUILDKIT=1 docker build -f install/docker/Dockerfile -t layer5/meshery --build-arg TOKEN=$(GLOBAL_TOKEN) --build-arg GIT_COMMITSHA=$(GIT_COMMITSHA) --build-arg GIT_VERSION=$(GIT_VERSION) --build-arg RELEASE_CHANNEL=${RELEASE_CHANNEL} --build-arg PROVIDER=$(LOCAL_PROVIDER) --build-arg PROVIDER_BASE_URLS=$(MESHERY_CLOUD_PROD) .
 
+## Build Meshery Server and UI container for e2e testing.
+docker-build-testing-env:
```
Comment (leecalcote) [inline Makefile:36]: Good! I wonder if we don't want to include these two new targets in the `.PHONY:` line so that the targets are readily invocable.

**Review by leecalcote — DISMISSED** [review]

File: install/docker/testing/Dockerfile
Comment (leecalcote) [inline install/docker/testing/Dockerfile:1]: Thank you for creating a new folder. 👍

File: Makefile (hunk 1 above)
Comment (jerensl) [reply Makefile]: Sure

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
Yay, @jerensl! 🎈

**Pull request merged**